packages/flutter_tools/lib/src/web/bootstrap.dart

```diff
@@ -267,15 +267,7 @@ $_simpleLoaderScript
             for (var i = 0; i < scripts.length; i++) {
               var script = scripts[i];
               if (script.id == null) continue;
-              var src = _currentDirectory + script.src.toString();
-              var oldSrc = window.\$dartLoader.moduleIdToUrl.get(script.id);
-
-              // We might actually load from a different uri, delete the old one
-              // just to be sure.
-              window.\$dartLoader.urlToModuleId.delete(oldSrc);
-
-              window.\$dartLoader.moduleIdToUrl.set(script.id, src);
-              window.\$dartLoader.urlToModuleId.set(src, script.id);
```

Comment:
Is it correct that we no longer need to update this map in case the script containing the module changes? Or are we saying that right now we are not supporting the module changing to a different file?

Reply:
The motivation here was that these were only used for tracking purposes and imports in the old DDC module format, so there isn't a real reason to track it unless we used the information meaningfully somehow. There is one meaningful corner-case we should be wary of:

https://github.com/dart-lang/sdk/blob/43b8d45e9b7dd44bacc7c221d11bb94367478f99/pkg/dev_compiler/lib/js/ddc/ddc_module_loader.js#L996

This checks that we don't accidentally load previously loaded modules in a call to `addScriptsToQueue`. This call occurs only during the bootstrapping (not during a hot restart or reload) but nothing really *prevents* us from calling it though.

So, to make this a little more resistant against such a possibility, I'm going to revert this clean-up.